docs/source/Tips-&-Tools.md

```diff
@@ -1,10 +1,6 @@
 
-* [[Foreman-Puppet]]
-* Credential for Red Hat package subscription 
-
-    > Username : mocsubsriber 
-
-    > Password : 451MOCTeamRoom
+* [Foreman-Puppet](clusters/Foreman-Puppet.html)
+* Credential for Red Hat package subscription: see bitwarden Redhat Packages Subsciption
   
 * [[Transitioning from Django to AngularJS]] 
```

Comment:
It looks like there are still a bunch of wiki-format `[[...]]` links in this document. Are those to be fixed in a future PR?